platform/ui-next/src/components/DataRow/types.ts

```diff
@@ -0,0 +1,13 @@
+export interface DataItem {
```

Comment:
type probably - this looks like a simple structure, not an interface for a class.